app/components/PurchaseTickets/Form.js

```diff
@@ -100,8 +99,8 @@ const PurchaseTicketsForm = ({
       <div className="stakepool-voting-title-area-name">
         <T id="purchaseTickets.title" m="Purchase Titckets" /></div>
       <div className="stakepool-purchase-ticket-input-buttons">
-        <PurchaseTicketsInfoButton onClick={onShowTicketsInfo}/>
-        <TicketsCogs opened={!isShowingAdvanced} onClick={onToggleShowAdvanced}/>
+        <Icon i="info" onClick={onShowTicketsInfo}/>
+        <Icon i="cog" pt={ 5 } pl={ 10 } active={isShowingAdvanced} onClick={onToggleShowAdvanced}/>
```

Comment:
pt and pl stand for "padding-top" and "padding-left"

padding and margin props accept numbers, strings, and arrays.

numbers 0-1 gets converted to percentage strings `1/2` to `"50%"`
integers < theme.space are theme.space[number]
larger numbers are converted to px strings

Strings are literal.

Arrays provide values for responsiveness set by breakpoints at theme.breakpoints.